src/main/java/com/github/webicitybrowser/spec/fetch/connection/imp/FetchConnectionImp.java

```diff
@@ -0,0 +1,58 @@
+package com.github.webicitybrowser.spec.fetch.connection.imp;
+
+import com.github.webicitybrowser.spec.fetch.Body;
+import com.github.webicitybrowser.spec.fetch.FetchRequest;
+import com.github.webicitybrowser.spec.fetch.FetchResponse;
+import com.github.webicitybrowser.spec.fetch.connection.FetchConnection;
+import com.github.webicitybrowser.spec.fetch.connection.FetchConnectionInfo;
+import com.github.webicitybrowser.spec.fetch.imp.FetchResponseImp;
+import com.github.webicitybrowser.spec.http.HTTPRequest;
+import com.github.webicitybrowser.spec.http.HTTPService;
+import com.github.webicitybrowser.spec.http.response.HTTPResponse;
+import com.github.webicitybrowser.spec.http.response.HTTPSuccessResponse;
+import com.github.webicitybrowser.webicity.core.net.ProtocolContext;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.InputStreamReader;
+
+public class FetchConnectionImp implements FetchConnection {
+
+	private final FetchConnectionInfo info;
+	private final HTTPService httpService;
+	private final static Logger logger = LoggerFactory.getLogger(FetchConnectionImp.class);
```

Comment:
Static fields should be at the very top (and with a blank line between them and the instance fields)